lib/utils.js

```diff
@@ -10,7 +11,81 @@ const delay = function(ms) {
   return new Promise(resolve => setTimeout(() => resolve(), ms));
 };
 
+/**
+ * Checks if a provided address is actively listening for incoming connections
+ *
+ * @param {string} host the host to connect to
+ * @param {number} port the port to check for availability
+ * @param {function} callback
+ */
+const checkAvailable = (host, port, callback) => {
```

Comment:
can we make this and `waitForAvailable` a function declaration instead of a function declaration? It will make it a lot easier to identify in a debugger.

Reply:
yup okay